lms/djangoapps/learner_home/mock_data.json

```diff
@@ -40,44 +40,42 @@
                 "progressUrl": "edx.com/courses/my-course-url/progress",
                 "unenrollUrl": "edx.com/courses/my-course-url/unenroll",
                 "resumeUrl": "edx.com/courses/my-course-url/resume",
-                "courseId": "course-number0-course-id0",
-                "lastEnrolled": "2022-09-06T15:20:57.513Z"
```

Comment:
Why remove?

Reply:
`lastEnrolled` got moved from `courseRun` to `enrollment`